src/hedera/hederaTypes.ts

```diff
@@ -8,6 +8,12 @@ export interface HederaSettings {
   maxFee: number
 }
 
+export const hederaOtherMethodNames = [
+  'getActivationSupportedCurrencies' as const,
+  'getActivationCost' as const,
+  'validateAccount' as const
+]
```

Comment:
The otherMethodsNames value for Hedera, EOS, and FIO are all typed a little different. The one for EOS makes the most sense I think.